app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/representee/confirm_person.scala.html

```diff
@@ -0,0 +1,101 @@
+@import play.twirl.api.Html
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.IndividualUserType.{Capacitor, PersonalRepresentative}
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.representee.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.RepresenteeAnswers.IncompleteRepresenteeAnswers
+
+@this(
+mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
+formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
+caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+summary_for_confirmation: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.returns.representee.summary_for_confirmation,
+radioGroupConditional: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.radio_group_conditional,
+errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
+backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
+yesNo: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.yes_no,
+textInput: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.text_input
+)
+
+@(
+        answers: IncompleteRepresenteeAnswers,
+        representativeType: Either[PersonalRepresentative.type, Capacitor.type],
+        displayReturnToSummaryLink: Boolean,
+        form: Form[Boolean],
+        backLink: Call
+)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
+@title = @{
+    messages("representeeConfirmPerson.title")
+}
+@key = @{
+    "confirmed"
+}
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+@formTitle = @{messages("representeeConfirmPerson.formTitle")}
+
+@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
+    @backLinkComponent(backLink)
+
+    @if(hasErrors) {
+        @errorSummary(form)
+    }
+    @caption(messages("representee.caption"))
+
+    @pageHeading(title)
+
+    @formWrapper(routes.RepresenteeController.confirmPersonSubmit(), 'novalidate -> "novalidate") {
+        @yesNo(
+            fieldId = key,
+            label = Html(s"<h2>$formTitle</h2>"),
+            labelAsHeading = true,
+            errorKey = form.error(key).map(e => e.message),
+            hasErrors = form.value.isEmpty,
+            selected = form.value,
+            helpText = None
+        )
+
+        @submitButton(messages("button.saveAndContinue"))
+    }
+
+    @summary_for_confirmation(answers, representativeType)
```

Comment:
this should probably go above the yes/no input - did you check what it looked like when you an it locally and compared it to the screenshot in the ticket?

Reply:
Yes, I had done it that way, but in PR I was asked to put it down.

Reply:
I think in my original comment I meant that only the return to summary link should go at the bottom of the page

Reply:
Done.